### DISCUSSION TITLE
Broken updates due to not adhering to semver, how to proceed?

### DISCUSSION THREAD

**Discussion by jrappen** [npm/feedback#924] [Registry]
The `vuepress` package's updates are broken.

Compare: <https://github.com/vuepress/vuepress-next/issues/1195>

tldr: They messed up a release by not adhering to `semver`, breaking all current and future beta releases.

Could you contact someone internally to get the broken release tags on npm removed? I understand that's the only current way to resolve this. They apparently haven't had luck contacting your support via your website according to comments on other issues in the repo linked above.

> Full disclosure:
>
> I'm related to neither VueJs, nor Vuepress, nor NPM, nor Github, nor Microsoft.

**Comment by ljharb** [comment]
The solution is them to publish a patch that fixes it, not to remove the tags from npm.
